docs/monitoring/metrics.md

```diff
@@ -543,62 +546,75 @@ Thus, in order to infer the metric identifier:
+      <th class="text-left" style="width: 20%">Metrics</th>                           
+      <th class="text-left" style="width: 32%">Description</th>
+      <th class="text-left" style="width: 8%">Type</th>                       
     </tr>                                                          
   </thead>                                                         
   <tbody>                                                          
     <tr>                                                           
       <th rowspan="12"><strong>Job-/TaskManager</strong></th>
       <td rowspan="12">Status.JVM.Memory</td>
       <td>Heap.Used</td>
-      <td>The amount of heap memory currently used.</td>
+      <td>The amount of heap memory currently used (in bytes).</td>
+      <td>Gauge</td>
     </tr>
     <tr>
       <td>Heap.Committed</td>
-      <td>The amount of heap memory guaranteed to be available to the JVM.</td>
+      <td>The amount of heap memory guaranteed to be available to the JVM (in bytes).</td>
+      <td>Gauge</td>
     </tr>
     <tr>
       <td>Heap.Max</td>
-      <td>The maximum amount of heap memory that can be used for memory management.</td>
+      <td>The maximum amount of heap memory that can be used for memory management (in bytes).</td>
+      <td>Gauge</td>
     </tr>
     <tr>
       <td>NonHeap.Used</td>
-      <td>The amount of non-heap memory currently used.</td>
+      <td>The amount of non-heap memory currently used (in bytes).</td>
+      <td>Gauge</td>
     </tr>
     <tr>
       <td>NonHeap.Committed</td>
-      <td>The amount of non-heap memory guaranteed to be available to the JVM.</td>
+      <td>The amount of non-heap memory guaranteed to be available to the JVM (in bytes).</td>
+      <td>Gauge</td>
     </tr>
     <tr>
       <td>NonHeap.Max</td>
-      <td>The maximum amount of non-heap memory that can be used for memory management.</td>
+      <td>The maximum amount of non-heap memory that can be used for memory management (in bytes).</td>
+      <td>Gauge</td>
     </tr>
     <tr>
       <td>Direct.Count</td>
-      <td>The number of buffers in the direct buffer pool.</td>
+      <td>The number of buffers in the direct buffer pool (in bytes).</td>
+      <td>Gauge</td>
     </tr>
     <tr>
       <td>Direct.MemoryUsed</td>
-      <td>The amount of memory used by the JVM for the direct buffer pool.</td>
+      <td>The amount of memory used by the JVM for the direct buffer pool (in bytes).</td>
+      <td>Gauge</td>
     </tr>
     <tr>
       <td>Direct.TotalCapacity</td>
-      <td>The total capacity of all buffers in the direct buffer pool.</td>
+      <td>The total capacity of all buffers in the direct buffer pool (in bytes).</td>
+      <td>Gauge</td>
     </tr>
     <tr>
       <td>Mapped.Count</td>
-      <td>The number of buffers in the mapped buffer pool.</td>
+      <td>The number of buffers in the mapped buffer pool (in bytes).</td>
```

Comment:
the unit is not bytes, but just a simple count.

Reply:
will change